Automattic/vip-go-mu-plugins
# Don't force 2FA for Jetpack SSO users that have Two-step enabled

## Description

When you you login via SSO, we set a cookie. Then we look for that
cookie to determine whether you logged in via SSO. We set an additional
cookie if your WordPress.com account has Two-step enabled.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Ensure you're not proxied because it is considered a second factor and therefore hides the 2FA admin notice
1. Login with Jetpack SSO (with Two-step enabled)
1. Ensure 2FA is not enabled
1. Confirm the 2FA admin notice is not present


1. Check out PR
1. Ensure you're not proxied
1. Login with a username and password
1. Ensure 2FA is not enabled
1. Confirm the 2FA admin is present
